File: CallDemo/CallDemo/ViewController.m
//
//  ViewController.m
//  CallDemo
//
//  Created by alexiuce  on 2017/7/20.
//  Copyright © 2017年 alexiuce . All rights reserved.
//

#import "ViewController.h"
#import "RegistController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)viewDidAppear:(BOOL)animated{
    [super viewDidAppear:animated];
    RegistController *regVC = [[RegistController alloc]init];
    [self presentViewController:regVC animated:YES completion:nil];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: CAShapLayerDemo/CAShapLayerDemo/ViewController.swift
//
//  ViewController.swift
//  CAShapLayerDemo
//
//  Created by alexiuce  on 2017/7/24.
//  Copyright © 2017年 alexiuce . All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var imgView: CustomImageView!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        
        if imgView.loadingView.progress > 1 {return}
        imgView.loadingView.progress += 0.1
        if imgView.loadingView.progress == 1.0 {
            imgView.loadingView.reavl()
        }
    }


}


